tests/Unit/API/Site/Controllers/Ads/AccountControllerTest.php

```diff
@@ -214,4 +215,14 @@ public function test_get_billing_status() {
 		$this->assertEquals( self::TEST_BILLING_STATUS_DATA, $response->get_data() );
 		$this->assertEquals( 200, $response->get_status() );
 	}
+
+	public function test_get_ads_accoount_has_access() {
```

Comment:
Small typo in the function name
```suggestion
	public function test_get_ads_account_has_access() {
```